packages/skeleton-react/src/lib/components/Progress/Progress.tsx

```diff
@@ -5,7 +5,7 @@ import { ProgressProps } from './types.js';
 export const Progress: FC<ProgressProps> = ({
 	value,
 	max = 100,
-	labelledBy = '',
+	labelledby = '',
```

Comment:
@bartduisters we don't call it out anywhere (yet), but we opt for camel-case for multi-word props names that *we* control. I realize it's confusing when compared to the raw ARIA attribute which is all lowercase though.

To further confuse things, event handlers also differ per framework:

- Svelte 5: `onclick` (lowercase)
- React: `onClick` (camel-case)

For context, this adheres to a change in Svelte v5. In prior versions they used `on:click`.

In practice they look like this:

## Svelte

```html
<!-- handler -->
<button onclick={doSomething}>
<!-- forwarded -->
<button onclick>
```

## React

```jsx
<button onClick={doSomething}>
```

> NOTE: can't recall if React allows for forwarding or not off hand.

Reply:
I initially changed it to camel case. But then noticed that on the Switch component it was all lowercase and thought that that was the convention.

I'll change the others then and add it to this PR